### PR TITLE
Support Unary Operator on Column

### PR DESCRIPTION
**Pull request by Quasimewdo**
- Make table in test_binaryExpr.py temporary
- support not(), abs(), pos(), neg()
- Add tests
- TODO :
  * Integrate unary operator in table.__getitem__
  * Probably need to change how to do unary_operator_column name alias

### CODE REVIEW

File: greenplumpython/expr.py [hunk 1]
```diff
@@ -64,6 +76,23 @@ def __str__(self) -> str:
...
+        super().__init__(as_name=as_name)
+        self.operator = operator
+        self.right = right
+
+    def __str__(self) -> str:
+        if self.operator == "NOT":
+            return "NOT(" + str(self.right) + ') AS "Not(' + str(self.right) + ')"'
+        if self.operator == "ABS":
+            return "ABS(" + str(self.right) + ') AS "Abs(' + str(self.right) + ')"'
+
```
Comment (beeender) [inline greenplumpython/expr.py:90]: we need to raise an exception for the unsupported operators. And, don't forget to add a test to check the exception.

File: greenplumpython/expr.py [hunk 2]
```diff
@@ -64,6 +76,23 @@ def __str__(self) -> str:
         return str(self.left) + " " + self.operator + " " + str(self.right)
 
 
+class UnaryExpr(Expr):
+    def __init__(self, operator: str, right: Expr, as_name: Optional[str] = None):
+        super().__init__(as_name=as_name)
+        self.operator = operator
+        self.right = right
+
+    def __str__(self) -> str:
```
Comment (beeender) [inline greenplumpython/expr.py:85]: Why do you add `As "xxxx"` in this function? I think the `as_name` should be handled by the parent class `Expr`?

Comment (Quasimewdo) [reply greenplumpython/expr.py:107]: Changed it to use `as_name`

File: tests/test_unaryExpr.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    assert str(b3) == '-temp3.id AS "-temp3.id"'
+    ret = list(t[["id", str(b3)]].fetch())
+    for row in ret:
+        assert row["-temp3.id"] == -(row["id"])
+
+
+def test_expr_unary_abs(db: gp.Database):
+    rows = [(1,), (-2,), (-3,), (2,)]
+    t = gp.values(rows, db=db).save_as("temp4", column_names=["id"], temp=True)
+    b4 = abs(t["id"])
```
Comment (xuebinsu) [inline tests/test_unaryExpr.py:48]: How about
```python
b4 = abs(t["id"]).rename("abs")
```

**Review by xuebinsu — APPROVED** [review]

**Review by beeender — APPROVED** [review]
:clap:

**Pull request merged**